src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -4,6 +4,7 @@
 
 using System;
 using System.Diagnostics;
+using System.Linq;
```

Comment:
please don't use linq in this layer.

Reply:
May I add a method like this: `private static bool All<T>(IReadOnlyList<T> items, Func<T, bool> predicate)`  or you prefer a local iteration? (there are two "All" usages)

Reply:
Don't use an IReadOnlyList as that will box.  These are SyntaxList's right?  So a helper that operates on those would be preferred, thanks!